src/Tests/AzureClientMagicTests.cs

```diff
@@ -39,6 +40,12 @@ public class AzureClientMagicTests
         private readonly string operationName = "TEST_OPERATION_NAME";
         private readonly string targetId = "TEST_TARGET_ID";
 
+        private readonly string jobParams = "{\"TEST_KEY1\":\"TEST_VAL1\",\"TEST_PATH\":\"C:\\ABC\\XYZ\"}";
```

Comment:
This may be easier to read in terms of C#'s `@""` strings.

Reply:
Good point, the `@` notation does look cleaner.  Also changed line 47 for consistency.